.env.example

```diff
@@ -2,3 +2,5 @@ COPILOT_API_KEY=""
 POSTGRES_PRISMA_URL=""
 POSTGRES_URL_NON_POOLING=""
 BLOB_READ_WRITE_TOKEN=""
+COPILOT_ENV="local"
+API_URL="http://localhost:3000/"
```

Comment:
@aatbip I think we replaced this with VERCEL_URL, so we can remove this

Reply:
I have removed this in the latest commit